volatility3/framework/layers/intel.py

```diff
@@ -265,7 +265,7 @@ class Intel32e(Intel):
     _direct_metadata = collections.ChainMap({'architecture': 'Intel64'}, Intel._direct_metadata)
     _entry_format = "<Q"
     _bits_per_register = 64
-    _maxphyaddr = 52
+    _maxphyaddr = 44
```

Comment:
This ideally wouldn't be applied to the whole Intel32e space, but instead just to the `WindowsIntel32e` class, since it might otherwise unnecessarily (and inaccurately) limit the architecture when it's more a windows limit.  I also wonder whether this should be handled in a way similar to the swap (so we allow the page fault to occur, but then catch and return the right value for it)?  I'm not sure how windows actually does it, and I'm not sure how they'd get around artificially limiting the architecture so high up (if there's a need for flags, why use bit 44 or 45 for the flag?).